setup-buildengine.sh

```diff
@@ -523,9 +530,11 @@ Creating $OPT_VM, compression=$OPT_COMPRESSION
  Release:     $OPT_RELEASE
  MerSDK VDI:  $OPT_VDI
  Tooling:     $OPT_TOOLING
- ARM target:  $OPT_TARGET_ARM
- i486 target: $OPT_TARGET_I486
 EOF
+for targetarch in ${OPT_TARGET_ARCHES[*]} ; do
```

Comment:
```suggestion
for targetarch in ${!OPT_TARGETS[*]}; do
```

Similarly elsewhere.

Reply:
OPT_TARGET_FILENAMES